### DISCUSSION TITLE
How to use The Energest Module

### DISCUSSION THREAD

**Discussion by r-leelavathi** [contiki-ng/contiki-ng#2408] [Q&A]
Hi all,
I tried implementing RPL in contiki-ng/ cooja. Simulation is working fine showing Avf Power Consumption graph and all. I need help in assiting how to use energest module in my simulation. Please guide me. I have used contiki-ng/example/ipv6/rpl-collect/upd-sink.c for sink node and udp-sender.c for 10 motes

**Comment by r-leelavathi** [comment]
I have used contiki-ng/example/rpl-udp/udp-client.c and udp-server.c.

**Comment by arurke** [comment]
Have you seen the Energest module tutorial: https://docs.contiki-ng.org/en/develop/doc/tutorials/Energy-monitoring.html#the-energest-module ?

**Comment by r-leelavathi** [comment]
Thank you for the reply. Is there a way to look at 'collect view' of a node  in cooja emulator, as it was in contiki while using RPL-collect?
